### PR TITLE
feature 1 pull 1

### PR DESCRIPTION
**Pull request by govindthange**
pls accept feature 1 pull 1 request

### CODE REVIEW

**Comment by AmarKoli** [comment]
can somebody review this code?

**Comment by AmarKoli** [comment]
anybody out there?

File: README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
 # techsoftwin
 
 www.techsoftwin.com
+
+feature 1
```
Comment (govindthange) [inline README.md:5]: Is this feature tested?

Comment (govindthange) [inline README.md:5]: also add feature 1.1.

File: README.md [hunk 2]
```diff
@@ -4,3 +4,5 @@ www.techsoftwin.com
 
 feature 1
 
+
+feature 1.1
```
Comment (AmarKoli) [inline README.md:8]: also add 1.2

**Comment by AmarKoli** [comment]
every thing looks fine. Merging now.

**Pull request merged**